tools/base/fpgainfo/fpgaerr.py

```diff
@@ -285,6 +286,13 @@ def args(self, parser):
                             default='fme',
                             help='specify what kind of errors to operate on')
 
+        # FIXME: should get a '-f' option later
```

Comment:
Seems like a easy FIXME to knock out now

Reply:
"-f" is currently used for the PCI function; as part of consolidating all tools, that should be turned into "-F", but that would change the current usage; we'll need to coordinate that carefully.